### PR TITLE
[Wiki] Update 'Contribution Guide' to latest

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Description

- Update 'Contribution Guide' to latest

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Update to documentation

### CODE REVIEW

File: docs/wiki/Contribution guide - Contribution flow.md [hunk 1]
```diff
@@ -53,7 +51,6 @@ How you proceed from here, depends on your scenario:
 - In case your contribution involves changes to the library (the modules) and/or to the CI environment (the pipelines), we kindly ask you to first review the [Module design](./The%20library%20-%20Module%20design) and/or the [Pipeline design](./The%20CI%20environment%20-%20Pipeline%20design) sections respectively, to make sure your contribution complies with the repository's design and principles.
 
 - If your contribution involves the creation of a new module or an update to an existing module:
-  - Make sure you also reflect your changes in the module's `README.md` file. Refer to the [Generate module Readme](./Contribution%20guide%20-%20Generate%20module%20Readme) section for details on how to leverage the Readme generator utility to support you with this task.
```
Comment (eriqua) [inline docs/wiki/Contribution guide - Contribution flow.md:56]: Is it intended to remove this step completely?

Comment (AlexanderSehr) [reply docs/wiki/Contribution guide - Contribution flow.md]: No. Just added it again.

File: docs/wiki/Contribution guide - Contribution flow.md [hunk 2]
```diff
@@ -66,18 +65,18 @@ Once the contribution is implemented and the changes are pushed to your forked r
...
-# Open a PR
+# Open a Pull Request (PR)
 
 Finally, once your contribution is implemented and validated, open a PR for the CARML maintainers to review. Make sure you:
 
-- Provide a meaningful PR title in the following format: `[<Category>] <PR title>`. For `[<Category>]` use one of the following, depending on which one is the primary improvement your PR intends to make:
+- Provide a meaningful title in the following format: `[<Category>] <PR title>`. For `[<Category>]` use one of the following, depending on which one is the primary improvement your PR intends to make:
   - `[Modules]`: For improvements to the CARML library.
   - `[CI environment]`: For improvements to the CI environment pipelines.
   - `[Static Validation]`: For improvements to Pester tests.
```
Comment (eriqua) [inline docs/wiki/Contribution guide - Contribution flow.md:75]: We eventually always went with` CI environment` instead
```suggestion
```

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**